diff --git a/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js b/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js
index 1ff7e89..f49a96b 100644
--- a/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js
+++ b/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js
@@ -91,6 +91,11 @@ class LinuxTargetHelper {
       sources.push(icnsPath);
     } // need to put here and not as default because need to resolve image size
     
+    // https://github.com/electron-userland/electron-builder/pull/5385
+    // if no explicit sources are defined, default to buildResources directory
+    if (sources.length < 1) {
+      sources.push('./')
+    }
 
     const result = await packager.resolveIcon(sources, (0, _builderUtil().asArray)(packager.getDefaultFrameworkIcon()), "set");
     this.maxIconPath = result[result.length - 1].file;
