diff --git a/node_modules/rcedit/lib/rcedit.js b/node_modules/rcedit/lib/rcedit.js
index 476a055..f77844f 100644
--- a/node_modules/rcedit/lib/rcedit.js
+++ b/node_modules/rcedit/lib/rcedit.js
@@ -7,7 +7,7 @@ const singleSettings = ['file-version', 'product-version', 'icon', 'requested-ex
 const noPrefixSettings = ['application-manifest']
 
 module.exports = async (exe, options) => {
-  let rcedit = path.resolve(__dirname, '..', 'bin', process.arch === 'x64' ? 'rcedit-x64.exe' : 'rcedit.exe')
+  let rcedit = path.resolve(__dirname, '..', 'bin', (process.arch === 'x64' || process.arch === 'arm64') ? 'rcedit-x64.exe' : 'rcedit.exe')
   const args = [exe]
 
   for (const name of pairSettings) {
