diff --git a/node_modules/tmp/lib/tmp.js b/node_modules/tmp/lib/tmp.js
index b41c29d..16d91b4 100644
--- a/node_modules/tmp/lib/tmp.js
+++ b/node_modules/tmp/lib/tmp.js
@@ -663,7 +663,13 @@ function setGracefulCleanup() {
  * @returns {string} the currently configured tmp dir
  */
 function _getTmpDir(options) {
-  return path.resolve(_sanitizeName(options && options.tmpdir || os.tmpdir()));
+  // os.tmpdir() sometimes doesn't return real path
+  // especially if user modifies TEMP/TMP env and add paths
+  // causing bug: EINVAL: invalid argument, mkdir 'C:\Users\username\AppData\Local\Temp;SystemDrive=C:\tmp-4344-G7jPrh6sY211'
+  // see https://github.com/nodejs/node/issues/11422
+  // see https://github.com/desktop/desktop/issues/3072
+  // see https://stackoverflow.com/questions/6521277/why-does-temp-resolve-to-a-non-deterministic-path-of-the-form-temp-digit
+  return path.resolve(_sanitizeName(options && options.tmpdir || os.tmpdir().split(';')[0]));
 }
 
 // Install process exit listener
